src/content/zero-knowledge-proofs.md

```diff
@@ -0,0 +1,177 @@
+
+Trusted setups require users to trust the participants in parameter-generation. However, the development of ZK-STARKs has enabled proving protocols that work with a non-trusted setup. 
+
+#### ZK-STARKs {#zk-starks}
+
+ZK-STARK is an acronym for **Zero-Knowledge Scalable Transparent Argument of Knowledge**. ZK-STARKs are similar to ZK-SNARKs, except that they are:
+
+- **Scalable**: ZK-STARK is faster than ZK-SNARK at generating and verifying proofs when the size of the witness is larger. With STARK proofs, prover and verification times only slightly increase as the witness grows (SNARK prover and verifier times increase linearly with witness size). 
+
+- **Transparent**: ZK-STARK relies on publicly verifiable randomness to generate public parameters for proving and verification instead of a trusted setup. Thus, they are more transparent compared to ZK-SNARKs. 
+
+ZK-STARKs produce larger proofs than ZK-SNARKs meaning they generally have higher verification overheads. However, there are cases (such as proving large datasets) where ZK-STARKs may be more cost-effective than ZK-SNARKs. 
+
+## Use-cases for zero-knowledge proofs {#use-cases-for-zero-knowledge-proofs}
+
+### Anonymous payments {#anonymous-payments} 
+
+Credit card payments are often visible to multiple parties, including the payments provider, banks, and other interested parties (e.g., government authorities). While financial surveillance has benefits for identifying illegal activity, it also undermines the privacy of ordinary citizens.
+
+Cryptocurrencies were intended to provide a means for users to conduct private, peer-to-peer transactions. But most cryptocurrency transactions are openly visible on public blockchains. User identities are often pseudonymous and either wilfully linked to real-world identities (e.g. by including ETH addresses on Twitter or Github profiles) or can be associated with real-world identities using basic on and off-chain data analysis. 
+
+There are specific “privacy coins” designed for completely anonymous transactions. Privacy-focused blockchains, such as ZCash and Monero, shield transaction details, including sender/receiver addresses, asset type, quantity, and the transaction timeline.
+
+By baking in zero-knowledge technology into the protocol, privacy-focused blockchain networks allow nodes to validate transactions without needing to access transaction data. 
+
+Zero-knowledge proofs are also being applied to anonymizing transactions on public blockchains. An example is Tornado Cash, a decentralized, non-custodial service that allows users to conduct private transactions on Ethereum. Tornado Cash uses zero-knowledge proofs to obfuscate transaction details and guarantee financial privacy.  Unfortunately, because these are "opt-in" privacy tools they are associated with illicit activity. To overcome this, privacy has to eventually become the default on public blockchains.
```

Comment:
> Unfortunately, because these are "opt-in" privacy tools they are associated with illicit activity. To overcome this, privacy has to eventually become the default on public blockchains.

It may be worth mentioning a note here about plausible deniability. By placing privacy at the application level, the simple fact that you're using that application  can raise unwanted attention, and it's difficult to deny that you're using it for that reason or that the privacy matters when using something like Torando cash, which has the sole purpose of making funds difficult to track. As you state, placing privacy as the default overcomes this.

Reply:
It's acc a good point.  For example, Tornado Cash is usually in the news each time some hacker is trying to launder stolen funds (e.g., what happened with the Harmony One token bridge hack). That said, I think @jmcook1186's description may be enough--talking  about plausible deniability may be too much info imo.